compiler-plugin-test/src/test/resources/test-src/package_03/file_service.bal

```diff
@@ -21,7 +21,7 @@ listener file:Listener localFolder = new ({
     recursive: false
 });
 
-service "filesystem" on localFolder {
+service file:Service "filesystem" on localFolder {
```

Comment:
AFAIK we don't need this with the latest lang fix

Reply:
Can you with the latest lang version

Reply:
It should be `ballerinaLangVersion=2.0.0-beta.4-20211117-131600-4133d87c`

Reply:
Updated